### PR TITLE
Added YAML, JSON, python files for assimilating VIIRS satwinds

### PR DESCRIPTION
**Pull request by BrettHoover-NOAA**
Adding satwinds from the Visible Infrared Imaging Radiometer Suite (VIIRS) from SNPP/NOAA-20 to GDASApp end-to-end testing

new files include:
parm/atm/obs/config/satwind_viirs_npp.yaml.j2: QC filter YAML for VIIRS SNPP satwinds (jinja2 standard)
parm/atm/obs/config/satwind_viirs_n20.yaml.j2: QC filter YAML for VIIRS NOAA-20 satwinds (jinja2 standard)
parm/ioda/bufr2ioda/bufr2ioda_satwind_amv_viirs.json: JSON containing data format, sensor, and satellite information for VIIRS SNPP/NOAA-20 satwinds
ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_viirs.py: bufr2ioda code for extracting VIIRS SNPP/NOAA-20 satwinds from BUFR

See #1054 for end-to-end testing results. Acceptance and ob-errors agree well with some expected deviation. No thinning is applied to these winds by regular convention.

Note: We are still using `qualityInformationWithoutForecast` as the variable-name for QI in the IODA converter. This variable-name is currently not registered in the IODA ObsSpace.yaml, but there is an ongoing discussion with JCSDA to have it added, issue is here: https://github.com/JCSDA-internal/ioda/issues/1233

### CODE REVIEW

**Comment by emilyhcliu** [comment]
@BrettHoover-NOAA Thanks.   Based on global_convinfo.txt  The AMVs from VIIRS (Type 260 with satids 224, 225, and 226)  have no thinning applied. (Yeh~).  
I am checking out your branch and will do a test run.  Thanks!!

File: ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_viirs.py [hunk 1]
```diff
@@ -0,0 +1,470 @@
...
+            obsspace.write_attr('Converter', converter)
+            obsspace.write_attr('sourceFiles', bufrfile)
+            obsspace.write_attr('dataProviderOrigin', data_provider)
+            obsspace.write_attr('description', data_description)
+            obsspace.write_attr('datetimeReference', reference_time)
+            obsspace.write_attr('datetimeRange', [str(timestamp2_min), str(timestamp2_max)])
+            obsspace.write_attr('sensor', sensor_id)
+            obsspace.write_attr('platform', satellite_id)
+            obsspace.write_attr('platformCommonName', satellite_name)
+            obsspace.write_attr('sensorCommonName', sensor_name)
```
Comment (nicholasesposito) [inline ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_viirs.py:337]: This attribute is not in the ObsSpace.yaml. I'd suggest adding it to the yaml though, as this information could be important.

File: ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_viirs.py [hunk 2]
```diff
@@ -0,0 +1,470 @@
...
+            obsspace.write_attr('processingLevel', process_level)
+            obsspace.write_attr('platformLongDescription', platform_description)
+            obsspace.write_attr('sensorLongDescription', sensor_description)
```
Comment (nicholasesposito) [inline ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_viirs.py:340]: This attribute is not in the ObsSpace.yaml, but it could also be worth adding.

**Review by nicholasesposito — COMMENTED** [review]
Some comments regarding some of the attributes, but otherwise look fine.

File: ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_viirs.py (hunk 2 above)
Comment (emilyhcliu) [reply ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_viirs.py:340]: @nicholasesposito  These names in the global attributes were created based on the IODA convention table.   But, you are right.  They are not in IODA [convention table](https://docs.google.com/spreadsheets/d/e/2PACX-1vRvM7vLTsUZs-xJZHha1vZDQZyFSrybKfNvN9lIFJY6yB2MQDmBH_MofWha2R7uhPl6frKKbGDQmd0g/pubhtml) (or ObsSpace.yaml)
JCSDA must change the naming convention for global attributes.  I will find out.

**Review by ADCollard — APPROVED** [review]
Looks good.  @nicholasesposito 's comment should not delay approval IMO.

**Review by PraveenKumar-NOAA — APPROVED** [review]
There are a lot of commented lines in both the YAMLs that can be removed, otherwise looks good.

**Review by nicholasesposito — APPROVED** [review]

**Comment by nicholasesposito** [comment]
> Looks good. @nicholasesposito 's comment should not delay approval IMO.

Approved on my end

**Pull request merged**